app/jobs/warm_up_facility_group_analytics_cache_job.rb

```diff
@@ -0,0 +1,12 @@
+class WarmUpFacilityGroupAnalyticsCacheJob < ApplicationJob
+  queue_as :default
+
+  def perform(facility_group, from_time_string, to_time_string)
```

Comment:
It's better to pass in IDs rather than full ruby objects to Jobs typically [1]. The ID then can be queried from within the job itself.

[1] https://github.com/mperham/sidekiq/wiki/Best-Practices#1-make-your-job-parameters-small-and-simple

Reply:
The activejob guide says it's safe to pass ActiveRecord objects to jobs, since they are passed as GlobalIds https://guides.rubyonrails.org/active_job_basics.html#globalid 

I can change it simply accept an ID if you think that's more explicit.

Reply:
Yes! GlobalID is one of those nice things in Rails 5 :)